python/ingestion/graphql_ahr_utils.py

```diff
@@ -33,11 +32,30 @@ def generate_cols_map(prefixes: list[str], suffix: str):
     'Diabetes': 'diabetes_per_100k',
     'Excessive Drinking': 'excessive_drinking_per_100k',
     'Frequent Mental Distress': 'frequent_mental_distress_per_100k',
-    'Preventable Hospitalizations': 'preventable_hospitalizations_per_100k',
     'Non-Medical Drug Use - Past Year': 'non_medical_drug_use_per_100k',
+}
+
+AHR_MEASURES_TO_RATES_MAP_ALL_AGES = {
```

Comment:
I think that the suicide measure is ages 15+

Reply:
i can't figure out this for sure